crates/router/src/types/transformers.rs

```diff
@@ -591,14 +591,18 @@ impl TryFrom<domain::MerchantConnectorAccount> for api_models::admin::MerchantCo
         };
         let frm_configs = match item.frm_configs {
```

Comment:
Move this logic to FRM crate and call it like

```
let frm_configs = frm::utils::get_frm_configs(req.frm_configs);
```

Reply:
this is different
previously we were trying to convert FrmConfigs type to value type 
here we are doing the reverse 
we cam keep this